### PR TITLE
Add recommendation in zha.markdown to upgrade ConBee and RaspBee

### PR DESCRIPTION
**Pull request by Hedda**
## Proposed change

Add recommendation in zha.markdown to upgrade deCONZ firmware (Zigbee stack application firmware) for ConBee and RaspBee as per https://github.com/dresden-elektronik/deconz-rest-plugin/releases ("This firmware update is also recommended for ZHA, zigbee2mqtt and WebThings users. In this release it needs to be installed manually as described in the [Wik](https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Update-deCONZ-manually)i").

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: NA
- Link to parent pull request in the Brands repository: NA
- This PR fixes or closes issue: NA

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/zha.markdown [hunk 1]
```diff
@@ -49,9 +49,9 @@ Some other Zigbee coordinator hardware may not support a firmware that is capabl
...
-  - [ConBee II (a.k.a. ConBee 2) USB adapter from dresden elektronik](https://phoscon.de/conbee2)
+  - [ConBee II (a.k.a. ConBee 2) USB adapter from dresden elektronik](https://phoscon.de/conbee2) (Note! Not a must but recommend [upgrade the deCONZ application firmware for ConBee II](https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Update-deCONZ-manually))
   - [ConBee USB adapter from dresden elektronik](https://phoscon.de/conbee)
-  - [RaspBee II (a.k.a. RaspBee 2) Raspberry Pi Shield from dresden elektronik](https://www.dresden-elektronik.com/product/raspbee-II.html)
+  - [RaspBee II (a.k.a. RaspBee 2) Raspberry Pi Shield from dresden elektronik](https://www.dresden-elektronik.com/product/raspbee-II.html)  (Note! Not a must but recommend [upgrade the deCONZ application firmware for ConBee II](https://github.com/dresden-elektronik/deconz-rest-plugin/wiki/Update-deCONZ-manually))
   - [RaspBee Raspberry Pi Shield from dresden elektronik](https://phoscon.de/raspbee)
 - Silicon Labs EmberZNet based radios using the EZSP protocol (via the [bellows](https://github.com/zigpy/bellows) library for zigpy)
```
Comment (frenck) [inline source/_integrations/zha.markdown:56]: I'm really sorry, but the list of supported radio modules has become unreadable list. We can just keep adding notes and links to each item.

**Review by frenck — CHANGES_REQUESTED** [review]
We should reconsider how to format the list and where to place radio specific instructions. This is growing out of control.

File: source/_integrations/zha.markdown (hunk 1 above)
Comment (Hedda) [reply source/_integrations/zha.markdown:56]: Maybe just a separate "Firmware" section further down with all those links? Or could we have subpages to integrations?

**Comment by Hedda** [comment]
> We should reconsider how to format the list and where to place radio specific instructions. This is growing out of control.

Maybe just add a separate "Firmware" section further down with all those links? Or could we have subpages to integrations?

EDIT: IMHO; independent support for Zigbee hardware adapters from different manufacturers is the key-feature of the ZHA integration so keep listing compatibility is very important if not the main point with ZHA that make it different from others.

Firmware version matter for compatibility and hardware related bug-fixes and update firmware is recommended but they can be hard to find so IMHO it is a good idea to help the user by URL linking to where to find firmware for upgrading their hardware.

**Comment by frenck** [comment]
> IMHO; independent support for Zigbee hardware adapters from different manufacturers is the key-feature of the ZHA integration so keep listing compatibility is very important if not the main point with ZHA that make it different from others.

I agree, however, making unreadable documentation has the same problem; as the user will not absorb the information.

I'm not adding this in at this point.

**Pull request closed without merging**